packages/manager/apps/pci-public-ip/src/pages/order/steps/FloatingIpSummary.tsx

```diff
@@ -44,16 +51,26 @@ export const FloatingIpSummary = ({
     networkId,
   );
 
-  const { addons: defaultGateways } = useAddons({
+  const useAddonsParams = {
```

Comment:
Params object must not begin with `use..` as it's a specific semantic convention for hooks in React.